rgw/v2/tests/s3_swift/test_storage_policy.py

```diff
@@ -144,6 +147,9 @@ def test_exec(config):
             umgmt.create_tenant_user(tenant_name=tenant, user_id=user_names[0],
                                      displayname=user_names[0])
             user_info = umgmt.create_subuser(tenant_name=tenant, user_id=user_names[0])
+            log.info(type(user_info))
```

Comment:
I think you don't need to log the type of the object here. can you remove this log message?